cql/ManageRareAbnormality.cql

```diff
@@ -1346,6 +1512,12 @@ define RecommendationForHistologyResults:
       null
   end
 
+define bothTreatmentBiopsy:
```

Comment:
Should this be removed? It looks like it's not used anywhere.

Reply:
Not used - will remove.

Reply:
Removed in commit https://github.com/ccsm-cds-tools/ccsm-cds-with-tests/pull/118/commits/c2704894cf2fdb29944fdfe2ff110d3d4094bda7.